.github/workflows/release-desktop.yml

```
@@ -59,6 +75,11 @@ jobs:
           path: packages/frontend/apps/electron/resources/web-static
 
   make-distribution:
+    if: >-
+      ${{
```

actionlint (1.7.9)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)